### PR TITLE
Update usrbg.css

### PR DESCRIPTION
**Pull request by VanillaSpace**
changed the GIF for my custom popout BG

### CODE REVIEW

File: dist/usrbg.css [hunk 1]
```diff
@@ -363,7 +363,8 @@
 
 [user_by_bdfdb*="227648235097817089"],
 [aria-label*="227648235097817089,"] {
-	--user-background: url('https://66.media.tumblr.com/69be13d66ffe5b85b4f75a9e2d544db6/tumblr_pu0pe5p27C1rsdpaso1_500.gif');
+	--user-popout-position: middle !important;
```
Comment (JonnyBoy2000) [inline dist/usrbg.css]: should be `center` not middle

Comment (VanillaSpace) [reply dist/usrbg.css]: okay

**Comment by Tropix126** [comment]
Also it defaults to center, so you dont need to put anything if it's center.

**Comment by VanillaSpace** [comment]
okay, in that case, I wouldn't need to say center then lol

**Comment by Tropix126** [comment]
Create a new pull request so that I can merge a working version please :)

**Comment by VanillaSpace** [comment]
okay will do, sorry I'm still a bit new to this

**Pull request closed without merging**